bridge_core.py

```
@@ -934,15 +1016,40 @@ def start_lights(self, num_lights):
+
         for i, light in enumerate(lights):
             try:
                 if isinstance(light, MultiZoneLight) and self.mz_startlights_mode == "sweep":
                     zone_count = self.get_zone_count(light)
                     if zone_count > 0:
-                        red_s  = list(red);  red_s[2]  = self._scale_brightness(red[2])
-                        dark_s = list(dark); dark_s[2] = self._scale_brightness(dark[2])
+                        # per_light recolours the whole strip by its label (per_zone
+                        # is handled per-zone below); default red otherwise.
+                        sweep_red = _override_hue_sat(red, _pl.get(self.safe_label(light))) if _do_pl else red
+                        red_s  = list(sweep_red); red_s[2]  = self._scale_brightness(sweep_red[2])
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)